terraform/modules/aws-network-load-balancer/locals.port-mapping.tf

```diff
@@ -0,0 +1,14 @@
+locals {
+  port_mapping = {
```

Comment:
We probably won't change these too often - I'd prefer writing the down this explicitly where we use it.

Reply:
I don't follow. Can you elaborate, pelase?

Reply:
The code fragments below loop over this `port_mapping` local variable. I think we could reduce a lot of complexity by just making it explicit there. It's a loop over two elements, not too much boilerplate, and makes things more readable.